plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -163,15 +171,21 @@ public CachingHiveMetastoreBuilder() {}
         private CachingHiveMetastoreBuilder(
                 HiveMetastore delegate,
                 Optional<Executor> executor,
+                boolean metadataCacheEnabled,
+                boolean statsCacheEnabled,
```

Comment:
why not keep `partitionCacheEnabled` together with these two?
i.e. it makes most sense to have `partitionCacheEnabled` just right after `statsCacheEnabled`

Reply:
Let's keep it where it is. I've already moved it once :)

Keeping these two separate reduces chance of passing wrong boolean